helm-chart/kuberay-operator/templates/deployment.yaml

```diff
@@ -37,11 +37,22 @@ spec:
           command:
             - /manager
           args:
-          {{- if .Values.batchScheduler.enabled }}
-            - --enable-batch-scheduler
-          {{- else }}
-            []
-          {{- end }}
+            {{- $argList := list -}}
+            {{- if .Values.batchScheduler.enabled -}}
+            {{- $argList = append $argList "--enable-batch-scheduler" -}}
+            {{- end -}}
+            {{- $watchNamespace := "" -}}
+            {{- if .Values.singleNamespaceInstall -}}
+            {{- $watchNamespace = .Release.Namespace -}}
+            {{- else if .Values.watchNamespace -}}
+            {{- $watchNamespace = .Values.watchNamespace -}}
+            {{- end -}}
+            {{- if $watchNamespace -}}
+            {{- $argList = append $argList "--watch-namespace" -}}
```

Comment:
`--watch-namespace` or `"-watch-namespace"?
 
https://github.com/ray-project/kuberay/blob/1d3f53721de17f6d5fa0910643c0c5909d32c0dd/manifests/overlays/single-namespace/patch-deployment.json#L5

Reply:
both formats work (I stuck with `--` because I've tested more thoroughly with it)